ts/a11y/explorer/KeyExplorer.ts

```diff
@@ -1293,13 +1293,48 @@ export class SpeechExplorer
   }
 
   /**
-   * Get an element's first speech child.
+   * Get an element's first speech child. This is computed by going through the
+   * owns list until the first speech element is found.
    *
    * @param {HTMLElement} node   The parent element to get a child from
    * @returns {HTMLElement}      The first speech child of the node
    */
   protected firstNode(node: HTMLElement): HTMLElement {
-    return node.querySelector(nav) as HTMLElement;
+    const owns = node.getAttribute('data-semantic-owns');
+    if (!owns) {
+      return node.querySelector(nav) as HTMLElement;
+    }
+    const ownsList = owns.split(/ /);
+    for (const id of ownsList) {
+      const node = this.getNode(id);
+      if (node?.hasAttribute('data-speech-node')) {
+        return node;
+      }
+    }
+    return null;
```

Comment:
Should this return `node.querySelector(nav)` rather than `null`?  I'm not sure.

Reply:
That works, too.